home.admin/00settingsMenuServices.sh

```diff
@@ -254,6 +254,17 @@ choice="off"; check=$(echo "${CHOICES}" | grep -c "pa")
 if [ ${check} -eq 1 ]; then choice="on"; fi
 if [ "${BTCPayServer}" != "${choice}" ]; then
   echo "BTCPayServer setting changed .."
+
+  #4049 warn if system has less than 8GB RAM
+  ramGB=$(free -g | awk '/^Mem:/{print $2}')
+  if [ "${choice}" =  "on" ] && [ ${ramGB} -lt 7 ]; then
+    whiptail --title "8GB RAM recommended for BTCPayServer" --yesno "8GB of RAM is recommended to run BTCPayServer on RaspiBlitz - you have less than that.\nDo you want to continue?" 10 50 --defaultno --yes-button "Continue" --no-button "Cancel"
```

Comment:
```suggestion
    whiptail --title "Your RaspiBlitz has less than the recommended 8GB of RAM to run BTCPayServer.\nDo you really want to proceed?" 10 50 --defaultno --yes-button "Continue" --no-button "Cancel"
```
Sounds better IMO. The `you have less than that` sounds like an accusation. Also inserted a `really` in the 2nd sentence to discourage more from continuing.